.gitignore

```diff
@@ -1,7 +1,6 @@
 HELP.md
 .gradle/
 build/
-local-dev/
```

Comment:
I don't think we want to remove this - local-dev usually holds clones of helm + config repos that shouldn't be tracked in git.
If it's to track the postgres script from this PR, I'd recommend explicitly including the script with `!`